README.md

```diff
@@ -1,2 +1,43 @@
-# Interstellar_Project
-This repository contains the pipelines used to analyze the data sets from the Interstellar collaboration.
+# Pyroptosis signature project 
+
+This repository contains the pipelines used to analyze the "waves" of data for the Interstellar collaboration project.
+"Waves" in the context of this repository means different parts of a 384 well plate that contain different stains, conditions, etc.
+
+## Background
+
+[Pyroptosis](https://www.nature.com/articles/nrmicro2070) is inflammatory cell death which occurs in response to pathogens and damage-associated molecular patterns.
+
+## Aims
+
+1. Describe an organelle morphology signature that predicts pyroptosis.
+2. Reveal the organelle interaction landscape during pyroptosis at an ultrastructural level.
```

Comment:
What does "ultrastructural" mean in this context? Consider providing a reference if it makes sense.

Reply:
@gwaybio Can you confirm the context here, I can only assume.

Reply:
@d33bs I can confirm it is referencing electron microscopy, so I had added that.